src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -31,6 +32,10 @@ private static void setupPresetTests(Application application) {
 
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
 		application.addTest("Figure Joe 2", selectTestFigureOptionListener);
+		application.addTest("Square", e -> FiguresFactory.square(-100, -100, 200)
+				.execute(DriverFeature.getDriverManager().getCurrentDriver()));
+		application.addTest("Triangle", e -> FiguresFactory.triangle(-50, 50, 100)
+				.execute(DriverFeature.getDriverManager().getCurrentDriver()));
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)